src/advanced_inv.cpp

```diff
@@ -1422,7 +1044,7 @@ bool advanced_inventory::move_all_items( bool nested_call )
 bool advanced_inventory::show_sort_menu( advanced_inventory_pane &pane )
 {
     uilist sm;
-    sm.text = _( "Sort by…" );
+    sm.text = _( "Sort by�" );
```

Comment:
You have invalid encoding here:

```suggestion
    sm.text = _( "Sort by…" );
```